### PR TITLE
Update documentation on op-version

### PR DESCRIPTION
**Pull request by samikshan**
Update op-version related documentation to include information on
the following:
 - How to get maximum supported op-version in a cluster
 - How to get op-version information for clients

Also updated the usage wrt getting current op-version to reflect
the new usage format to query global options using volume get all.

### CODE REVIEW

File: Upgrade-Guide/op_version.md [hunk 1]
```diff
@@ -10,7 +10,11 @@ After Gluster upgrade, it is advisable to have op-version updated.
 
 Current op-version can be queried as below:
 
-    [root@~]#gluster volume get  <volume name> cluster.op-version
+    [root@~]#gluster volume get all cluster.op-version
+
+To check maximum op-version supported by the cluster, the following query can be used:
```
Comment (ShyamsundarR) [inline Upgrade-Guide/op_version.md]: This should be, "to get maximum op-version that the cluster can support" right? I mean is one node can support X+ and another X then this returns X.

File: Upgrade-Guide/op_version.md [hunk 2]
```diff
@@ -20,3 +24,9 @@ For example, after upgrading to glusterfs-3.7.1, set op-version as:
 Note: 
 This is not mandatory, but advisable to have updated op-version if you want to make use of latest features in the updated gluster.
 
+###Client op-version
+
+To check op-version information for clients the following query can be used:
+
+    [root@~]#gluster volume status <all|VOLNAME> clients
```
Comment (ShyamsundarR) [inline Upgrade-Guide/op_version.md]: Can we add some text to explan how this helps a user? For example, get this information and the cluster max op-version information and either set it to a minimum of these 2, or upgrade the clients, or what else?

Comment (samikshan) [inline Upgrade-Guide/op_version.md]: "When trying to set a volume option, it might happen that one or more of the connected clients cannot support the feature being set and might need to be upgraded to the op-version the cluster is currently running on.

To check op-version information for the connected clients and find the offending client, the following query can be used:

    [root@~]#gluster volume status <all|VOLNAME> clients

The respective clients can then be upgraded to the required version.

This information can also be used to make an informed decision while bumping up the op-version of a cluster, so that connected clients can still support all the new features provided by the upgraded cluster."

Is this better?

File: Upgrade-Guide/op_version.md (hunk 1 above)
Comment (samikshan) [inline Upgrade-Guide/op_version.md]: I felt this meant the same though. But I do see how it could be confusing. Will reword this to make it clearer.

**Comment by prashanthpai** [comment]
I'd also love to see a table (could be another PR) on gluster releases and their op-versions. This will be similar to how it's captured in [downstream doc](https://access.redhat.com/articles/2356261).
